biome.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "vcs": {
+    "clientKind": "git",
+    "enabled": true,
+    "useIgnoreFile": true
+  },
+  "files": {
+    "ignoreUnknown": true
```

Comment:
biome currently doesn't support a lot of extensions. You can see the list here https://biomejs.dev/internals/language-support/

This tells biome to ignore them instead of throwing an error